akka-http-caching/src/test/scala/akka/http/caching/ExpiringRefreshingCacheSpec.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2017-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.caching
+
+import java.util.concurrent.Executor
+import akka.actor.ActorSystem
+import akka.http.caching.scaladsl.CachingSettings
+import akka.testkit.TestKit
+import com.github.benmanes.caffeine.cache.AsyncCacheLoader
+import org.scalatest.BeforeAndAfterAll
+
+import scala.compat.java8.FutureConverters.{ toJava => toJavaFuture }
+import scala.concurrent.duration._
+import scala.concurrent.{ Await, Future }
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class ExpiringRefreshingCacheSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll {
+  implicit val system: ActorSystem = ActorSystem()
+  import system.dispatcher
+
+  "A RefreshingCache" should {
+    "refresh values after the refresh-after-write" in {
+      val wait = 1.second
```

Comment:
4s for 4 test cases is a lot in the whole test suite.

Would be good to tune this down quite a bit and share a single wait time for all tests and the config rather than repeat. I think something like 200ms should be fine.